lib/govspeak.rb

```diff
@@ -62,6 +62,7 @@ def initialize(source, options = {})
                    sanitize: true,
                    syntax_highlighter: nil }.merge(options)
       @options[:entity_output] = :symbolic
+      @footnote_definition_html = ""
```

Comment:
We appear to be using an empty string to replicate nil-ish behaviour here.
It would probably be better to set this to `nil` initially, and change line 76 to `unless @footnote_definition_html.nil?`.

Reply:
Thanks @ChrisBAshton , I've updated this now - see [here](https://github.com/alphagov/govspeak/pull/216/files#diff-acb6db6cd67285bd5380398c9e51ced8940ed7c4110706d038cbb243310e4055R65) and [here](https://github.com/alphagov/govspeak/pull/216/files#diff-acb6db6cd67285bd5380398c9e51ced8940ed7c4110706d038cbb243310e4055R77)